pkg/operator/ceph/file/mds/spec.go

```diff
@@ -44,8 +44,8 @@ const (
 func (c *Cluster) makeDeployment(mdsConfig *mdsConfig, namespace string) (*apps.Deployment, error) {
 
 	mdsContainer := c.makeMdsDaemonContainer(mdsConfig)
-	mdsContainer = config.ConfigureStartupProbe(cephv1.KeyMds, mdsContainer, c.clusterSpec.HealthCheck)
-	mdsContainer = config.ConfigureLivenessProbe(cephv1.KeyMds, mdsContainer, c.clusterSpec.HealthCheck)
+	cephconfig.ConfigureStartupProbe(mdsContainer, c.fs.Spec.StatusCheck.StartupProbe)
```

Comment:
Need to assign the result back to the container wherever these helpers are called, right?
```suggestion
	mdsContainer = cephconfig.ConfigureStartupProbe(mdsContainer, c.fs.Spec.StatusCheck.StartupProbe)
```

Reply:
Thanks, added it